src/config/dynamic-theme-fixes.config

```diff
@@ -1260,6 +1260,19 @@ img.listview__image, img.galleryview__image {
 
 ================================
 
+geogebra.org
+
+INVERT
+.icon-m
+.elemText canvas
+.gwt-Image
+.gwt-StackPanelItem img
+.GeoGebraMenuImage.menuImg
+.menuImg
+.buttonContent.stylebarButton
+
```

Comment:
Let's add `canvas[class^="cursor"]` So the canvas will also be targetted

Reply:
I tried to invert the main canvas but the lines were blinking when I moved around. The app supports change lines and background color in settings, that works well.

Reply:
I saw that when trying to just get the class `cursor_hit`
But when blinking it just changes the class after the _ so cursor_drag cursor_pen
So with CSS3 selectors we can target `startWith` with ^= so
canvas[class^="cursor"] means that it only target canvas with a class that start with cursor and then you don't have the blinking anymore :)

Reply:
I used the selector `.EuclidianPanel canvas` earlier, I think the results are the same. After adding a few functions, you will notice the blinking at some part of the lines you added when you move the mouse or drag the canvas.

Reply:
.EuclidianPanel > canvas
Add that and then it will be merged